ts/navigation/AppStackNavigator.tsx

```diff
@@ -169,6 +176,10 @@ const InnerNavigationContainer = (props: { children: React.ReactElement }) => {
   const cgnEnabled = useIOSelector(isCGNEnabledSelector);
   const isFimsEnabled = useIOSelector(isFIMSEnabledSelector) && fimsEnabled;
 
+  const bpdRemoteConfig = useIOSelector(bpdRemoteConfigSelector);
+  const isOptInPaymentMethodsEnabled =
+    bpdRemoteConfig?.opt_in_payment_methods && bpdOptInPaymentMethodsEnabled;
```

Comment:
Shouldn't this be `bpdRemoteConfig?.opt_in_payment_methods_v2 && bpdOptInPaymentMethodsEnabled` ?

Reply:
😮 nice catch! It was inherited from a CP&P 

fixed in [f083851](https://github.com/pagopa/io-app/pull/4164/commits/f083851fc38b119a2e0abd163c9ed6fc46447844)